source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/classic/cartpole/cartpole_camera_env_cfg.py

```diff
@@ -88,6 +88,38 @@ class DepthCameraPolicyCfg(RGBObservationsCfg.RGBCameraPolicyCfg):
     policy: ObsGroup = DepthCameraPolicyCfg()
 
 
+@configclass
+class ResNet18ObservationCfg:
+    """Observation specifications for the MDP."""
+
+    @configclass
+    class ResNet18FeaturesCameraPolicyCfg(RGBObservationsCfg.RGBCameraPolicyCfg):
+        """Observations for policy group with features extracted from RGB images with a frozen ResNet18."""
+
+        image = ObsTerm(
```

Comment:
Would not overwrite the parents. Is the inheritance from RgbCCmaeraPolicyCfg necessary?